### PR TITLE
HARP-5699 Add simple object demo

### PR DESCRIPTION
**Pull request by nzjony**
Demo which shows how to add a THREE.js object to the scene and position using RTC (relative to center).

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/harp.gl/450)

### CODE REVIEW

File: @here/harp-examples/src/hello_add_simple_object.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            // end:harp_gl_hello_add_simple_object_2.ts
+
+            // snippet:harp_gl_hello_add_simple_object_3.ts
+            // Add a callback to execute before the items are rendered.
+            map.addEventListener(MapViewEventNames.Render, () => {
+                // Set the cube position relative to the world center. Note, we don't subtract the
+                // [[worldCenter]] from the worldMousePosition, because we need to keep the cubes
+                // world position untouched.
+                cube.position.copy(worldPositionAtMouse);
+                cube.position.sub(map.worldCenter);
```
Comment (Astrak) [inline @here/harp-examples/src/hello_add_simple_object.ts]: you can chain `copy().sub()`

Comment (nzjony) [reply @here/harp-examples/src/hello_add_simple_object.ts]: thanks, awesome tip!

**Comment by musculman** [comment]
Why would you call it `hello_add_simple_object`? Wouldn't it be easier to have it as `add_threejs_simple_object`?
I believe that `hello_world` is probably fitting the first and hopefully easiest example we have, but once past that, there is no need to keep the `hello` suffix. Or what was the idea behind?

**Comment by nzjony** [comment]
@musculman, good question. I also discussed this with @ninok, we thought that because it is the equivalent of a hello world example with threejs, that it is probably ok, however true, it might be more appropriate to name it based on the fact that it is showing how to use threejs with the map.

I want to add an example showing raycasting, and we decided to call it threejs_raycast, how about calling this threejs_add_simple_object?

**Comment by ninok** [comment]
> I want to add an example showing raycasting, and we decided to call it threejs_raycast, how about calling this threejs_add_simple_object?
maybe call it threejs_addSimpleObject otherwise you get a lot of "subfolders"

**Comment by nzjony** [comment]
> threejs_addSimpleObject

fail enough regarding the sub folders, the convention, at least with the other example is to use the - to separate the words

**Pull request merged**